heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "heapoverflow",
+    "label": "HeapOverflow",
```

Add a description here that will display in the UI (feel free to edit),
```suggestion
    "label": "HeapOverflow",
    "description": "Demonstration of using heap overflow to achieve arbitrary code execution",
```